### ISSUE TITLE
nxos_lag_interfaces fails when force parameter is set 

### ISSUE THREAD

**Issue by Marcusz97** [ansible-collections/cisco.nxos#110]
##### SUMMARY
The nxos_lag_interfaces fails when the force parameter is set to either yes or no.

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME
nxos_lag_interfaces

##### ANSIBLE VERSION

```
ansible 2.9.10
  config file = /etc/ansible/ansible.cfg
  configured module search path = [u'/root/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python2.7/dist-packages/ansible
  executable location = /usr/bin/ansible
  python version = 2.7.17 (default, Apr 15 2020, 17:20:14) [GCC 7.5.0]
```

##### CONFIGURATION

```
DEFAULT_ASK_PASS(/Users/rchambers/dcnf/ansible.cfg) = True
DEFAULT_FORKS(/Users/rchambers/dcnf/ansible.cfg) = 10
DEFAULT_GATHERING(/Users/rchambers/dcnf/ansible.cfg) = explicit
DEFAULT_HOST_LIST(/Users/rchambers/dcnf/ansible.cfg) = ['/Users/rchambers/dcnf/fabric-lab.ini']
DEFAULT_MODULE_NAME(/Users/rchambers/dcnf/ansible.cfg) = command
HOST_KEY_CHECKING(/Users/rchambers/dcnf/ansible.cfg) = False
INTERPRETER_PYTHON(/Users/rchambers/dcnf/ansible.cfg) = auto_silent
PERSISTENT_COMMAND_TIMEOUT(/Users/rchambers/dcnf/ansible.cfg) = 30
PERSISTENT_CONNECT_RETRY_TIMEOUT(/Users/rchambers/dcnf/ansible.cfg) = 15
PERSISTENT_CONNECT_TIMEOUT(/Users/rchambers/dcnf/ansible.cfg) = 60
RETRY_FILES_ENABLED(/Users/rchambers/dcnf/ansible.cfg) = False

```

##### OS / ENVIRONMENT

Target host: Cisco Nexus NX-OSv, NXOS 9.3.3

##### STEPS TO REPRODUCE

```yaml
- name: add interface to the Port-channel
  nxos_lag_interfaces:
    config:
      - name: port-channel1
        members:
          - member: Ethernet1/4
            force: "true"
            mode: active
```

##### EXPECTED RESULTS

Expected "changed", member link added to port-channel.

##### ACTUAL RESULTS

```
TASK [interfaces : add interface to the Port-channel] **************************
task path: /etc/ansible/roles/interfaces/tasks/AddToPO.yml:2
<172.16.0.10> attempting to start connection
<172.16.0.10> using connection plugin network_cli
<172.16.0.10> local domain socket does not exist, starting it
<172.16.0.10> control socket path is /root/.ansible/pc/a623f43a53
<172.16.0.10> local domain socket listeners started successfully
<172.16.0.10> loaded cliconf plugin nxos from path /usr/lib/python2.7/dist-packages/ansible/plugins/cliconf/nxos.py for network_os nxos
<172.16.0.10>
<172.16.0.10> local domain socket path is /root/.ansible/pc/a623f43a53
<172.16.0.10> ESTABLISH LOCAL CONNECTION FOR USER: root
<172.16.0.10> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /root/.ansible/tmp/ansible-local-16539QGd1qI `"&& mkdir /root/.ansible/tmp/ansible-local-16539QGd1qI/an16546-126889472298623 && echo ansible-tmp-1593773244.37-16546-126889472298623="` echo /root/.ansible/tmp/ansible-local-16539QGd1qI/ansible-tmp-1593773244.37-16546-12eep 0'
<172.16.0.10> Attempting python interpreter discovery
<172.16.0.10> EXEC /bin/sh -c 'echo PLATFORM; uname; echo FOUND; command -v '"'"'/usr/bin/python'"'"'; command -v '"'"'python3.7'"'"'; command -v '"'"'python3.6'"'"'3.5'"'"'; command -v '"'"'python2.7'"'"'; command -v '"'"'python2.6'"'"'; command -v '"'"'/usr/libexec/platform-python'"'"'; command -v '"'"'/usr/bin/python3'"'"'; c"'; echo ENDFOUND && sleep 0'
<172.16.0.10> EXEC /bin/sh -c '/usr/bin/python && sleep 0'
Using module file /usr/lib/python2.7/dist-packages/ansible/modules/network/nxos/nxos_lag_interfaces.py
<172.16.0.10> PUT /root/.ansible/tmp/ansible-local-16539QGd1qI/tmpL1ZeGT TO /root/.ansible/tmp/ansible-local-16539QGd1qI/ansible-tmp-1593773244.37-16546-126889472298nterfaces.py
<172.16.0.10> EXEC /bin/sh -c 'chmod u+x /root/.ansible/tmp/ansible-local-16539QGd1qI/ansible-tmp-1593773244.37-16546-126889472298623/ /root/.ansible/tmp/ansible-locmp-1593773244.37-16546-126889472298623/AnsiballZ_nxos_lag_interfaces.py && sleep 0'
<172.16.0.10> EXEC /bin/sh -c '/usr/bin/python /root/.ansible/tmp/ansible-local-16539QGd1qI/ansible-tmp-1593773244.37-16546-126889472298623/AnsiballZ_nxos_lag_interf
<172.16.0.10> EXEC /bin/sh -c 'rm -f -r /root/.ansible/tmp/ansible-local-16539QGd1qI/ansible-tmp-1593773244.37-16546-126889472298623/ > /dev/null 2>&1 && sleep 0'
The full traceback is:
Traceback (most recent call last):
  File "/root/.ansible/tmp/ansible-local-16539QGd1qI/ansible-tmp-1593773244.37-16546-126889472298623/AnsiballZ_nxos_lag_interfaces.py", line 102, in <module>
    _ansiballz_main()
  File "/root/.ansible/tmp/ansible-local-16539QGd1qI/ansible-tmp-1593773244.37-16546-126889472298623/AnsiballZ_nxos_lag_interfaces.py", line 94, in _ansiballz_main
    invoke_module(zipped_mod, temp_path, ANSIBALLZ_PARAMS)
  File "/root/.ansible/tmp/ansible-local-16539QGd1qI/ansible-tmp-1593773244.37-16546-126889472298623/AnsiballZ_nxos_lag_interfaces.py", line 40, in invoke_module
    runpy.run_module(mod_name='ansible.modules.network.nxos.nxos_lag_interfaces', init_globals=None, run_name='__main__', alter_sys=True)
  File "/usr/lib/python2.7/runpy.py", line 188, in run_module
    fname, loader, pkg_name)
  File "/usr/lib/python2.7/runpy.py", line 82, in _run_module_code
    mod_name, mod_fname, mod_loader, pkg_name)
  File "/usr/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/tmp/ansible_nxos_lag_interfaces_payload_zAxJRu/ansible_nxos_lag_interfaces_payload.zip/ansible/modules/network/nxos/nxos_lag_interfaces.py", line 231, in <m
  File "/tmp/ansible_nxos_lag_interfaces_payload_zAxJRu/ansible_nxos_lag_interfaces_payload.zip/ansible/modules/network/nxos/nxos_lag_interfaces.py", line 226, in ma
  File "/tmp/ansible_nxos_lag_interfaces_payload_zAxJRu/ansible_nxos_lag_interfaces_payload.zip/ansible/module_utils/network/nxos/config/lag_interfaces/lag_interfacete_module
  File "/tmp/ansible_nxos_lag_interfaces_payload_zAxJRu/ansible_nxos_lag_interfaces_payload.zip/ansible/module_utils/network/nxos/config/lag_interfaces/lag_interfaceconfig
  File "/tmp/ansible_nxos_lag_interfaces_payload_zAxJRu/ansible_nxos_lag_interfaces_payload.zip/ansible/module_utils/network/nxos/config/lag_interfaces/lag_interfacestate
  File "/tmp/ansible_nxos_lag_interfaces_payload_zAxJRu/ansible_nxos_lag_interfaces_payload.zip/ansible/module_utils/network/nxos/config/lag_interfaces/lag_interfacete_merged
  File "/tmp/ansible_nxos_lag_interfaces_payload_zAxJRu/ansible_nxos_lag_interfaces_payload.zip/ansible/module_utils/network/nxos/config/lag_interfaces/lag_interfacecommands
  File "/tmp/ansible_nxos_lag_interfaces_payload_zAxJRu/ansible_nxos_lag_interfaces_payload.zip/ansible/module_utils/network/nxos/config/lag_interfaces/lag_interfacecommands
TypeError: cannot concatenate 'str' and 'bool' objects
[DEPRECATION WARNING]: Distribution Ubuntu 18.04 on host 172.16.0.10 should use
 /usr/bin/python3, but is using /usr/bin/python for backward compatibility with
 prior Ansible releases. A future Ansible release will default to using the
discovered platform python for this host. See https://docs.ansible.com/ansible/
2.9/reference_appendices/interpreter_discovery.html for more information. This
feature will be removed in version 2.12. Deprecation warnings can be disabled
by setting deprecation_warnings=False in ansible.cfg.
fatal: [172.16.0.10]: FAILED! => {
    "ansible_facts": {
        "discovered_interpreter_python": "/usr/bin/python"
    },
    "changed": false,
    "module_stderr": "Traceback (most recent call last):\n  File \"/root/.ansible/tmp/ansible-local-16539QGd1qI/ansible-tmp-1593773244.37-16546-126889472298623/Ansib.py\", line 102, in <module>\n    _ansiballz_main()\n  File \"/root/.ansible/tmp/ansible-local-16539QGd1qI/ansible-tmp-1593773244.37-16546-126889472298623/AnsiballZ_, line 94, in _ansiballz_main\n    invoke_module(zipped_mod, temp_path, ANSIBALLZ_PARAMS)\n  File \"/root/.ansible/tmp/ansible-local-16539QGd1qI/ansible-tmp-15937732623/AnsiballZ_nxos_lag_interfaces.py\", line 40, in invoke_module\n    runpy.run_module(mod_name='ansible.modules.network.nxos.nxos_lag_interfaces', init_globals=Nonalter_sys=True)\n  File \"/usr/lib/python2.7/runpy.py\", line 188, in run_module\n    fname, loader, pkg_name)\n  File \"/usr/lib/python2.7/runpy.py\", line 82, in __name, mod_fname, mod_loader, pkg_name)\n  File \"/usr/lib/python2.7/runpy.py\", line 72, in _run_code\n    exec code in run_globals\n  File \"/tmp/ansible_nxos_lag_u/ansible_nxos_lag_interfaces_payload.zip/ansible/modules/network/nxos/nxos_lag_interfaces.py\", line 231, in <module>\n  File \"/tmp/ansible_nxos_lag_interfaces_pay_lag_interfaces_payload.zip/ansible/modules/network/nxos/nxos_lag_interfaces.py\", line 226, in main\n  File \"/tmp/ansible_nxos_lag_interfaces_payload_zAxJRu/ansiblyload.zip/ansible/module_utils/network/nxos/config/lag_interfaces/lag_interfaces.py\", line 62, in execute_module\n  File \"/tmp/ansible_nxos_lag_interfaces_payload_interfaces_payload.zip/ansible/module_utils/network/nxos/config/lag_interfaces/lag_interfaces.py\", line 100, in set_config\n  File \"/tmp/ansible_nxos_lag_interface_nxos_lag_interfaces_payload.zip/ansible/module_utils/network/nxos/config/lag_interfaces/lag_interfaces.py\", line 122, in set_state\n  File \"/tmp/ansible_nxos_lag_u/ansible_nxos_lag_interfaces_payload.zip/ansible/module_utils/network/nxos/config/lag_interfaces/lag_interfaces.py\", line 166, in _state_merged\n  File \"/tmp/ansipayload_zAxJRu/ansible_nxos_lag_interfaces_payload.zip/ansible/module_utils/network/nxos/config/lag_interfaces/lag_interfaces.py\", line 243, in set_commands\n  File_interfaces_payload_zAxJRu/ansible_nxos_lag_interfaces_payload.zip/ansible/module_utils/network/nxos/config/lag_interfaces/lag_interfaces.py\", line 228, in add_commconcatenate 'str' and 'bool' objects\n",
    "module_stdout": "",
    "msg": "MODULE FAILURE\nSee stdout/stderr for the exact error",
    "rc": 1
}

PLAY RECAP *********************************************************************
172.16.0.10                : ok=0    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0
```

**Comment by Marcusz97** [comment]
I think the issue is at this line:

https://github.com/ansible/ansible/blob/v2.9.10/lib/ansible/module_utils/network/nxos/config/lag_interfaces/lag_interfaces.py#L228

I think the correct line is:

`cmd = group_cmd + ' force '`

or even better:
    
```python
if 'force' in d and d['force']:
                cmd = group_cmd + ' force '
```

**Comment by NilashishC** [comment]
Closed by #111
